config/initializers/ab_tests.rb

```diff
@@ -96,4 +96,14 @@ def self.all
         IdentityConfig.store.recommend_webauthn_platform_for_sms_ab_test_authentication_percent,
     },
   ).freeze
+
+  DESKTOP_FT_UNLOCK_SETUP = AbTest.new(
+    experiment_name: 'Desktop F/T unlock setup',
+    should_log: [
+      :webauthn_setup_submitted,
+      'Multi-Factor Authentication Setup',
+    ].to_set,
```

Comment:
For your queries in https://github.com/18F/identity-idp/pull/11347#discussion_r1852722922, this needs to include the event  'User Registration: 2FA Setup visited'